src/Middlewares/CacheResponse.php

```diff
@@ -31,6 +32,12 @@ public function handle(Request $request, Closure $next, ...$args): Response
 
                 $response = $this->responseCache->getCachedResponseFor($request, $tags);
 
+                // Add cache age header
```

Comment:
Could move this functionality in a function with a good name, so we can lose that comment?

Reply:
done!